### PR TITLE
impr(quotes): Added a few more quotes to the list!

### PR DESCRIPTION
**Pull request by CedricBaaklini**
I took the liberty to add a few quotes onto the JSON file for the English language from various pop culture media.

### CODE REVIEW

**Comment by Miodec** [comment]
Your PR includes a theme. Please remove unrelated changes.

File: frontend/static/quotes/english.json [hunk 1]
```diff
@@ -38640,6 +38640,36 @@
...
+      "text": "I am a monument to all your sins.",
+      "source": "Halo 2",
+      "length": 33,
+      "id": 7033
+    },
+    {
+      "text": "Nothing is true; everything is permitted.",
+      "source": "Assassin's Creed",
+      "length": 41,
+      "id": 7034
```
Comment (Miodec) [inline frontend/static/quotes/english.json:38672]: Also, these quotes are too short.

File: frontend/static/quotes/english.json [hunk 2]
```diff
@@ -38640,6 +38640,36 @@
...
+      "text": "The Truth. I am a god, boy, from another land, far from here. When I came to these shores, I chose to live as a man. But the truth is... I was born a god. And so were you.",
+      "source": "God of War (2018)",
+      "length": 171,
+      "id": 7030
+    },
+    {
+      "text": "Look at that subtle off-white coloring. The tasteful thickness of it. Oh, my God. It even has a watermark.",
+      "source": "American Psycho",
+      "length": 106,
+      "id": 7031
```
Comment (Miodec) [inline frontend/static/quotes/english.json:38654]: Inconsistent capitalisation in these quotes. 'God' is sometimes capitalised sometimes not. Why is 'truth' capitalised?

Comment (CedricBaaklini) [reply frontend/static/quotes/english.json:38654]: That was just how the script was written, but I can fix it if you like.

File: frontend/static/quotes/english.json [hunk 3]
```diff
@@ -38640,6 +38640,18 @@
...
+      "text": "I am a god, boy, from another land, far from here. When I came to these shores, I chose to live as a man. But the truth is... I was born a god. And so were you.",
+      "source": "God of War (2018)",
+      "length": 171,
+      "id": 7030
+    },
+    {
+      "text": "Look at that subtle off-white coloring. The tasteful thickness of it. Oh, my God. It even has a watermark.",
+      "source": "American Psycho",
+      "length": 106,
+      "id": 7031
```
Comment (Miodec) [inline frontend/static/quotes/english.json:38654]: Duplicate of quote `6562`. Please check for duplicates before adding quotes.

**Review by Miodec — CHANGES_REQUESTED** [review]

**Comment by monkeytypegeorge** [comment]
Continuous integration check(s) failed. Please review the failing check's logs and make the necessary changes. https://github.com/monkeytypegame/monkeytype/actions/runs/8003299821

**Review by Miodec — APPROVED** [review]

File: frontend/static/quotes/english.json [hunk 4]
```diff
@@ -38640,6 +38640,24 @@
...
+      "id": 7030
+    },
+    {
+      "text": "You don't have to be the bad guy. You are the most talented, most interesting, and most extraordinary person in the universe. And you are capable of amazing things. Because you are the Special. And so am I. And so is everyone. The prophecy is made up, but it's also true. It's about all of us. Right now, it's about you. And you... still... can change everything.",
+      "source": "The LEGO Movie",
+      "length":363,
+      "id": 7031
+    },
+    {
+      "text": "Rorschach’s Journal. October 12th, 1985: Dog carcass in alley this morning, tire tread on burst stomach. This city is afraid of me. I have seen its true face. The streets are extended gutters and the gutters are full of blood and when the drains finally scab over, all the vermin will drown. The accumulated filth of all their sex and murder will foam up about their waists and all the whores and politicians will look up and shout “Save us!”… and I’ll whisper “no.”",
```
Comment (Miodec) [inline frontend/static/quotes/english.json:38657]: It mentions a dead dog in an alley? Not the best choice.

Comment (CedricBaaklini) [reply frontend/static/quotes/english.json:38657]: Wait, you guys don't even want references of death?

Comment (Miodec) [reply frontend/static/quotes/english.json:38657]: Kids use this site, so i prefer to keep it without much details.

**Comment by Miodec** [comment]
Please merge master in and resolve conflicts.

**Comment by monkeytypegeorge** [comment]
Continuous integration check(s) failed. Please review the failing check's logs and make the necessary changes. https://github.com/monkeytypegame/monkeytype/actions/runs/8091815895

**Pull request closed without merging**